.changeset/unlucky-doors-cheer.md

```diff
@@ -0,0 +1,9 @@
+---
+"@patternfly/pfe-core": minor
+---
+
+✨ Added `ScrollSpyController`
```

Comment:
Need a changeset entry here for the addition of `roving-tabindex-controller` as well